crates/proto/src/rr/domain/usage.rs

```diff
@@ -580,6 +566,22 @@ impl ZoneUsage {
     }
 }
 
+/// Constructs a new Default, with all no restrictions
+impl Default for ZoneUsage {
+    fn default() -> Self {
+        Self::new(
+            Name::root(),
+            UserUsage::Normal,
+            AppUsage::Normal,
+            ResolverUsage::Normal,
+            CacheUsage::Normal,
+            AuthUsage::Normal,
+            OpUsage::Normal,
+            RegistryUsage::Normal,
+        )
+    }
+}
+
```

Comment:
On this one I wasn't sure, maybe it's just the name, you don't wanted to pull in all the implication of `Default`.

Reply:
this seems fine, I don't know why that default method wasn't pulled out into `Default`..